lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -22,6 +22,7 @@
 #include "glow/Graph/Nodes.h"
 #include "glow/IR/IRUtils.h"
 #include "glow/IR/Instrs.h"
+#include "glow/Quantization/Quantization.h"
```

Comment:
why does it need Quantization.h? Depending on Quantization lib will introduce circular dependency which was fixed recently :)